test/integration/api/access_tokens_test.rb

```diff
@@ -1,41 +1,89 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class Admin::Api::AccessTokensTest < ActionDispatch::IntegrationTest
   def setup
-    @provider = FactoryGirl.create(:provider_account)
+    @provider = FactoryGirl.create(:simple_provider, provider_account: master_account)
     @admin = FactoryGirl.create(:simple_admin, account: @provider)
     @admin.activate!
     @member = FactoryGirl.create(:simple_user, account: @provider)
 
     host! @provider.self_domain
   end
 
-  test 'create with access_token' do
-    access_token = FactoryGirl.create(:access_token, owner: @admin, scopes: %w(account_management))
+  test 'create with access_token can not create for another user' do
```

Comment:
We have them mostly for the representers, but it is true that we have a representer now and it doesn't have any test:
https://github.com/3scale/porta/blob/7ea57c2d7d949ee3b9665918747437d3bd5f4850/app/representers/access_token_representer.rb#L1-L15
Before it was not officially supported so ok, but now it will be officially supported so you are right and I will add it 👍

Reply:
Done 👍